package.json

```diff
@@ -47,28 +52,34 @@
     "stylint": "1.5.9"
   },
   "dependencies": {
-    "@atlaskit/editor-core": "134.0.1",
+    "@atlaskit/editor-core": "acezard/editor-core-fork",
```

Comment:
We should target a specific tag if we want to be able to rebuild a specific release in the future.

Reply:
Fork should be managed by cozy orgs, not your account

Reply:
@Crash-- absolutely, how shall we name it? just `cozy/cozy-editor-core` maybe?

Reply:
cozy/editor-core